src/js/component.js

```diff
@@ -1714,7 +1714,7 @@ class Component {
    */
   requestNamedAnimationFrame(name, fn) {
     if (this.namedRafs_.has(name)) {
-      return;
+      this.cancelNamedAnimationFrame(name);
```

Comment:
Previous, if a raf with that name is already queued, it returned and did not add the new raf of this name.

Now if the named raf is queued, instead of returning, that queued one is cancelled, then the new raf is added.